docs/src/modules/java/pages/kickstart.adoc

```diff
@@ -165,6 +165,55 @@ The Maven failsafe plugin runs the integration tests when the `it` profile is en
 mvn verify -Pit
 ----
 
+=== Unit Tests for Event Sourced Entities with autogenerated Testkit
+
+We can test an Event Sourced Entity implementation with the AkkaserverlessTestkit. This class is autogenerated by Akkaserverless when a project with an EventSourceEntity is compiled. Creating an ad-hoc TestKit. 
+
+For a service with the following RPCs, api and domain, it will generate a ShoppingCartTestKit. Here we only post a reduced version of the of the files of this project. The whole of it can be found in https://github.com/lightbend/akkaserverless-java-sdk/tree/main/samples/java-eventsourced-shopping-cart
+
+.src/main/proto/shoppingcart/shoppingcart_api.proto
+[source,java]
+----
+include::example$testkit/java-eventsourced-shopping-cart/src/main/proto/shoppingcart/shoppingcart_api.proto[] 
+----
+
+and the domain
+.src/main/proto/shoppingcart/shoppingcart_domain.proto
+[source,java]
+----
+include::example$testkit/java-eventsourced-shopping-cart/src/main/proto/shoppingcart/shoppingcart_domain.proto[] 
+----
+
+In the API we can see the service our ShoppingCart must implement is addItem and with it and ItemAdded must also be added as the shoppingcart_domain.proto shows. 
+
+Here we can see an implementation of the ShoppingCart
+
+.src/main/java/com/example/shoppingcart/ShoppingCart.java
+[source,java]
+----
+include::example$testkit/java-eventsourced-shopping-cart/src/main/java/com/example/shoppingcart/ShoppingCart.java[] 
+----
+
+To test this implementation with the autogenerate ShoppingCartTestKit, which can be found in your project in /target/generated-test-sources/akkaserverless/java/com/example/shoppingcart/domain/ShoppingCartTestKit.java
+
+This is a simplification of its contents but in essence what we show in the next snippet is the signatura of the available methods
+
+.target/generated-test-sources/com/example/shoppingcart/domain/ShoppingCartTestKit.java
+[source,java]
+----
+include::example$testkit/java-eventsourced-shopping-cart/target/generated-test-sources/akkaserverless/com/example/shoppingcart/domain/ShoppingCartTestKit.java[] 
+----
+
+Finally using the testkit to test our implementation abides to our requirement can be this. 
+
+.src/test/java/com/example/shoppingcart/domain/ShoppingCartTest.java
+[source,java]
+----
+include::example$testkit/java-eventsourced-shopping-cart/src/test/java/com/example/shoppingcart/domain/ShoppingCartTest.java[] 
```

Comment:
shouldn't it be `include::example$java-eventsourced-shopping-cart`?

Reply:
you're right. Done now